CHANGELOG.rst

```diff
@@ -15,6 +15,11 @@ Change Log
 
 Unreleased
 ----------
+[4.6.9]
+-------
+refactor: removing "null=true" constraint from encrypted fields
```

Comment:
[question] why did we feel the need to disable null values for this field?

Reply:
As per docs, we should be saving null values first https://django-fernet-fields.readthedocs.io/en/latest/#migrations

Reply:
@sameenfatima78 because the unencrypted fields are using default null=False